src/tools/pub/pubspec.md

```diff
@@ -269,6 +273,65 @@ For more information, see
 [pub global](/tools/pub/cmd/pub-global#running-a-script-from-your-path).
 
 
+### Platforms
+
+When publishing packages to the [pub.dev site.]({{site.pub}}) the set of
+_supported platforms_ will be automatically detected based on transitively
+imported `dart:...` libraries and [Flutter plugin declarations][]. The set of
+supported platforms is displayed on the package page, and used for filtering
+when searching for packages on the [pub.dev site.]({{site.pub}}).
+
+If the supported platforms automatically detected is incorrect, the list of
+_supported platforms_ can be explicitly declared using the `platforms:` key
+as follows:
+
+{% prettify yaml tag=pre+code %}
+# This package supports all platforms listed below
+platforms:
+  android:
+  ios:
+  linux:
+  macos:
+  web:
+  windows:
+{% endprettify %}
+
+The automatic detection of supported platforms is not perfect. For example, a
+package using `chmod` through `Process.run` from `dart:io` won't work on
+Windows, even though `dart:io` is available on Windows. Hence, it is reasonable
+to explicitly declare that the package only supports Linux and MacOS, as
+illustrated below:
+
+{% prettify yaml tag=pre+code %}
+# This package only supports Linux and MacOS
+platforms:
+  linux:
+  macos:
+{% endprettify %}
```

Comment:
````suggestion
```yaml
# This package only supports Linux and macOS
platforms:
  linux:
  macos:
```
````